templates/documentation/get-started/infra.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "api.video location and infrastructure"
```

Comment:
This title should be the same as the H1 - let's go with "api.video infrastructure"